flexbox/src/main/java/com/google/android/flexbox/FlexLine.java

```diff
@@ -95,6 +95,14 @@
     int mLastIndex;
 
     /**
+     * Indicates whether this flex line instance is in a dirty state (needs to be computed before
+     * rendering). This flag is set to true for example an item is added at the position not
+     * visible and the position is prior to the position than the first visible position.
```

Comment:
"and the position is prior to the position than the first visible position"
We are marking this as true when the newly added item comes after the last visible position, right?